dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hdfs/ScanContext.java

```diff
@@ -152,24 +176,22 @@ String getFormattedStats() {
             : Duration.ZERO;
 
     final long numFilesDivisor = numFiles > 0 ? numFiles : 1;
-    StringBuilder sb =
-        new StringBuilder()
-            .append("[HDFS Permission extraction stats]")
-            .append("\nTotal: num files&dirs: " + numFiles)
-            .append("\n       num dirs found: " + numDirs)
-            .append("\n       num dirs walkd: " + numDirsWalked)
-            .append("\nTotal File Size: " + accumulatedFileSize)
-            .append("\nAvg File Size: " + accumulatedFileSize / numFilesDivisor)
-            .append("\nTotal time: ")
-            .append(timeSinceScanBegin.getSeconds() + "s")
-            .append("\nTotal time in listStatus(..): ")
-            .append(timeSpentInListStatus.getSeconds() + "s")
-            .append("\nAvg time per file in listStatus(..): ")
-            .append(avgTimeSpentInListStatusPerFile.toMillis() + "ms")
-            .append("\nAvg time per doc: ")
-            .append(timeSinceScanBegin.dividedBy(numFilesDivisor).toMillis() + "ms\n")
-            .append("\n[/HDFS Permission extraction stats]");
-
-    return sb.toString();
+    return new StringBuilder()
+        .append("[HDFS Permission extraction stats]")
+        .append("\nTotal: num files&dirs: " + numFiles)
+        .append("\n       num dirs found: " + numDirs)
+        .append("\n       num dirs walked: " + numDirsWalked)
+        .append("\nTotal File Size: " + accumulatedFileSize)
+        .append("\nAvg File Size: " + accumulatedFileSize / numFilesDivisor)
+        .append("\nTotal time: ")
+        .append(timeSinceScanBegin.getSeconds() + "s")
+        .append("\nTotal time in listStatus(..): ")
+        .append(timeSpentInListStatus.getSeconds() + "s")
```

Comment:
This should probably be a chained append(), not a String-+ in the middle, as that creates an extra StringBuilder and String.